File: editorial/chapter-6-16_20/prob6-18.cpp
#include <iostream>
using namespace std;

long long GCD(long long A, long long B) {
	if (B == 0) return A;
	return GCD(B, A % B);
}

int main() {
	// “ü—Í
	long long A, B;
	cin >> A >> B;

	// Å¬Œö”{”‚ª 10^18 ‚ð’´‚¦‚é‚©‚Ç‚¤‚©”»’è
	if (A / GCD(A, B) > 1000000000000000000 / B) {
		cout << "Large" << endl;
	}
	else {
		cout << A / GCD(A, B) * B << endl;
	}
	return 0;
}

File: editorial/chap6-16_20/prob6-18.py
# ユークリッドの互除法を使って最大公約数を求めます（別の方法として、標準ライブラリ math.gcd(A, B) を使う手もあります）
def GCD(A, B):
	if B == 0:
		return A
	return GCD(B, A % B)

# 入力
A, B = map(int, input().split())

# 最小公倍数を求める（注：Python ではオーバーフローを気にする必要はありません）
L = A * B // GCD(A, B)
if L > 10 ** 18:
	print("Large")
else:
	print(L)